### PR TITLE
Garths synth changes

### PR DESCRIPTION
**Pull request by justinjmoses**
Here are your changes garth incorporated into a local branch.

I've made a couple of notes inline for you to review.

### CODE REVIEW

File: content/synopsis.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
...
-In principle the system can support any asset with a clear price, and provides on-chain exposure to an unlimited range of real-world assets.
+Synthetix is a protocol for issuing and trading synthetic assets on [Ethereum](https://www.ethereum.org/). Each synthetic asset (or [Synth](/tokens#synthetic-assets-synths)) is an [ERC20 token](https://theethereum.wiki/w/index.php/ERC20_Token_Standard) which tracks the price of an external asset; for example each [sUSD](https://etherscan.io/token/0x57ab1ec28d129707052df4df418d58a2d46d5f51) token tracks the price of the US dollar (and unlike the other synthetic assets, is fixed at `1`). A wide variety of Synths exists within Synthetix, including fiat currencies, cryptocurrencies, commodities, and [inverse indexes](/tokens#inverse-isynths). In principle the system can support any asset with a clear price, and provides on-chain exposure to an unlimited range of real-world assets.
```
Comment (justinjmoses) [inline content/synopsis.md:3]: I fixed the links to `/token` - if you run `mkdocs serve` locally you'll see when links are broken and can fix them accordingly.

File: content/synopsis.md [hunk 2]
```diff
@@ -1,10 +1,9 @@
...
 
-A more descriptive overview of the system is available in the [Synthetix Litepaper](/litepaper), and the explainers on [SNX staking rewards](https://www.synthetix.io/stakingrewards), [Synthetix.Exchange](https://www.synthetix.io/products/exchange), and [Mintr](https://www.synthetix.io/products/mintr). There is also a [community-run page](https://synthetix.community) full of helpful resources, and a snapshot of the current system status is always live at [dashboard.synthetix.io](https://dashboard.synthetix.io/).
+A more descriptive overview of the system is available in the [Synthetix Litepaper](/litepaper), and the explainers on [SNX staking rewards](?), [Synthetix.Exchange](https://www.synthetix.io/products/exchange), and [Mintr](https://www.synthetix.io/products/mintr). There is also a [community-run page](https://synthetix.community) full of helpful resources, and a snapshot of the current system status is always live at [dashboard.synthetix.io](https://dashboard.synthetix.io/).
```
Comment (justinjmoses) [inline content/synopsis.md:9]: What is the URL for the "Staking Rewards" link here?

File: scripts/build-tokens.js [hunk 3]
```diff
@@ -161,4 +161,4 @@ ${tokens
 	.join('\n\n')}
 
 `;
-fs.writeFileSync(path.join(__dirname, '..', 'content', 'tokens.md'), content);
+fs.writeFileSync(path.join(__dirname, '..', 'content', 'tokens', 'list.md'), content);
```
Comment (justinjmoses) [inline scripts/build-tokens.js:164]: This was the imperative change I needed to make on your PR - the auto-generated token list URL changed so I needed to make sure successive builds knew which path to overwrite

File: mkdocs.yml [hunk 4]
```diff
@@ -16,16 +16,17 @@ nav:
       - Incentives: incentives.md
       - Governance: governance.md
   - Litepaper:
-      - Home: litepaper/index.md
-      - Korean: litepaper/korean.md
-      - Russian: litepaper/russian.md
-      - Chinese: litepaper/chinese.md
+      - '🇦🇺 English ': litepaper/index.md
```
Comment (justinjmoses) [inline mkdocs.yml:19]: country flags ftw!

File: content/synopsis.md (hunk 2 above)
Comment (garthtravers) [reply content/synopsis.md:9]: Yeah I guess we've deprecated that page since.

Comment (justinjmoses) [reply content/synopsis.md:9]: OK so I'll remove it

File: mkdocs.yml (hunk 4 above)
Comment (garthtravers) [reply mkdocs.yml:19]: v nice

File: content/synopsis.md (hunk 1 above)
Comment (garthtravers) [reply content/synopsis.md]: I just added a few more links into this section (into each type of Synth)

**Pull request merged**